src/__internal__/input-icon-toggle/input-icon-toggle.component.tsx

```diff
@@ -83,11 +82,12 @@ const InputIconToggle = ({
         onFocus={onFocus}
         onBlur={onBlur}
         onMouseDown={onMouseDown}
-        tabIndex={iconTabIndex}
         data-element="input-icon-toggle"
         disabled={disabled}
         readOnly={readOnly}
         data-role="input-icon-toggle"
+        aria-hidden="true"
```

Comment:
Question (non-blocking): Are we certain that there's no use cases where we want it read by voiceover or be tabbable etc? If not then maybe it would be better as a props that default etc. Non-blocking for me as it's trivial to change at a later date if we need to